backends/fs/posix/posix-fs.cpp

```diff
@@ -257,7 +261,11 @@ bool POSIXFilesystemNode::create(bool isDirectoryFlag) {
 	bool success;
 
 	if (isDirectoryFlag) {
+#ifdef PSP2
+		success = sceIoMkdir(_path.c_str(), 0755) == 0;
```

Comment:
Did you consider making something like

`#define mkdir sceIoMkdir`

in your platform-specific OSystem definition file?